de-lib/src/main/java/org/iplantc/de/desktop/client/presenter/InitializationCallbacks.java

```diff
@@ -39,7 +41,7 @@
         private final DesktopPresenterImpl presenter;
         private final UserSessionServiceFacade userSessionService;
         private final UserPreferencesCallback userPreferencesCallback;
-        Logger LOG = Logger.getLogger(BootstrapCallback.class.getName());
+        Logger LOG = Logger.getLogger("BootstrapCallback");
```

Comment:
I could switch it back and try to just prioritize the deobfuscation ticket too... so either way.